app.js

```diff
@@ -3,88 +3,117 @@
-  },
-  cookiesPerHour: []
-};
-var capitolHill = {
-  name: 'capitolHill',
-  minCustomers:20,
-  maxCustomers:38,
-  avgCookies:2.3,
-  randomCustomers: function(){
-    return Math.floor(Math.random() * (38 - 20)) + 20;
-  },
-  cookiesPerHour: []
-};
-var alki = {
-  name: 'alki',
-  minCustomers:2,
-  maxCustomers:16,
-  avgCookies:4.6,
-  randomCustomers: function(){
-    return Math.floor(Math.random() * (16 - 2)) + 2;
-  },
-  cookiesPerHour: []
+function CookieStore(name, minCustomers, maxCustomers, avgCookies){
+  this.name = name;
+  this.minCustomers = minCustomers;
+  this.maxCustomers = maxCustomers;
+  this.avgCookies = avgCookies;
+  this.cookiesPerHour = [];
+  this.range = maxCustomers - minCustomers;
+}
+
+var firstAndPike = new CookieStore('First and Pike', 23, 65, 6.3);
+var seatacAirport = new CookieStore('Seatac Airport', 3, 24, 1.2);
+var seattleCenter = new CookieStore('Seattle Center', 11, 38, 3.7);
+var capitolHill = new CookieStore('Capitol Hill', 20, 38, 2.3);
+var alki = new CookieStore('Alki', 2, 16, 4.6);
+
+CookieStore.prototype.randomCustomers = function(){
+  return Math.floor(Math.random() * (this.range + 1) + this.minCustomers);
 };
 
 //This is a function that can be called for each store/object.
 var getCookiesPerHour = function(objectName){
-  for(var i = 0; i < 15; i++){
+  for(var i = 0; i < storeHours.length; i++){
     objectName.cookiesPerHour.push(Math.floor(objectName.randomCustomers() * objectName.avgCookies));
   }
 };
 
-//Call a function for each city to populate the arrays in each object.
-getCookiesPerHour(firstAndPike);
-getCookiesPerHour(seatacAirport);
-getCookiesPerHour(seattleCenter);
-getCookiesPerHour(capitolHill);
-getCookiesPerHour(alki);
-
-//Create function to create an ul and insert the array into it.
-function putSalesProjections(location){
-  console.log(location.name + '-projections');
-  var ulElement = document.getElementById(location.name + '-projections');
-
-  for(var ii = 0; ii < 15; ii++){
-    var liElement = document.createElement('li');
-    liElement.setAttribute('class', 'avgCookiesPerHour');
-    liElement.innerHTML = storeHours[ii] + ': ' + location.cookiesPerHour[ii] + ' cookies';
-    ulElement.appendChild(liElement);
+CookieStore.prototype.makeTheTable = function(tableToDrawInto){
+  var hourlySalesRowEl = document.createElement('tr');
+  console.log(hourlySalesRowEl);
+  var storeNameEl = document.createElement('th');
+  console.log(storeNameEl);
+  storeNameEl.textContent = this.name;
+  hourlySalesRowEl.appendChild(storeNameEl);
+
+  for(var hours = 0; hours < this.cookiesPerHour.length; hours++){
+    var hourlyCookieSales = document.createElement('td');
+    console.log(hourlyCookieSales);
+    hourlyCookieSales.textContent = this.cookiesPerHour[hours];
+    console.log(hourlyCookieSales);
+    hourlySalesRowEl.appendChild(hourlyCookieSales);
+  }
+  tableToDrawInto.appendChild(hourlySalesRowEl);
+};
+
+var tableEl = document.createElement('table');
```

Comment:
It'll be helpful for you if you group global variables like this up at the very top.